### PR TITLE
Add CoC and Contributing Guide to follow Community Standards

### PR DESCRIPTION
**Pull request by avsej**
The content was adopted from https://github.com/couchbase/couchbase-jvm-clients/commit/afa6fffc3c23b5603a2c6a722c1c33ed0edf5da0

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+We use Jira for issue tracking: [https://issues.couchbase.com/projects/CXXCBC/](https://issues.couchbase.com/projects/CXXCBC/)
+
+To create an issue, or comment on an existing issue, you'll need to sign up
+for an account. It's quick and painless (or if it's not, click on the
+"Report a problem" link on the signup page or get in touch via the forum).
+
+## Pull Requests
+
+We use Github for code reivew. But to allow us to integrate your changes, you need to register for a Gerrit account and
```
Comment (raycardillo) [inline CONTRIBUTING.md:23]: spell check: `reivew` -> `review`

**Review by raycardillo — CHANGES_REQUESTED** [review]
Looks good, but fix the typo first.

File: CONTRIBUTING.md (hunk 1 above)
Comment (avsej) [reply CONTRIBUTING.md]: thanks

**Review by raycardillo — COMMENTED** [review]
LGTM.

**Review by raycardillo — APPROVED** [review]
Approved.

**Comment by raycardillo** [comment]
Merged.

**Pull request merged**